pypeman/channels.py

```diff
@@ -237,36 +259,75 @@ def process(self, msg):
         else:
             return msg
 
+    @asyncio.coroutine
+    def replay(self, msg_id):
+        """
+        This method allow you to replay a message from channel `message_store`.
+        :param msg_id: Message id to replay
+        :return: the result of the processing.
+        """
+        msg_dict = self.message_store.get(msg_id)
+        new_message = msg_dict['message'].renew()
+        result = self.handle(new_message)
+        return result
+
     def graph(self, prefix='', dot=False):
         for node in self._nodes:
             if isinstance(node, SubChannel):
-                print(prefix + '|—\\')
+                print(prefix + '|—\\ (%s)' % node.name)
                 node.graph(prefix= '|  ' + prefix)
             elif isinstance(node, ConditionSubChannel):
-                print(prefix + '|?\\')
+                print(prefix + '|?\\ (%s)' % node.name)
                 node.graph(prefix='|  ' + prefix)
                 print(prefix + '|  -> Out')
+            elif isinstance(node, Case):
+                for i, c in enumerate(node.cases):
+                    print(prefix + '|c%s\\' % i)
+                    c[1].graph(prefix='|  ' + prefix)
+                    print(prefix + '|<--')
             else:
                 print(prefix + '|-' + node.name)
 
-    def graph_dot(self, previous='', end=''):
+    def graph_dot(self, end=''):
```

Comment:
always prints to stdout. shouldn't we do either:
print with a file parameter or just return list of strings, so the function can be reused